solr/test-framework/src/java/org/apache/solr/SolrTestCase.java

```diff
@@ -19,10 +19,12 @@
 
 import java.lang.invoke.MethodHandles;
 import java.io.File;
+import java.util.regex.Pattern;
 
 import com.carrotsearch.randomizedtesting.annotations.ThreadLeakLingering;
 import org.apache.lucene.util.LuceneTestCase;
 import org.apache.lucene.util.QuickPatchThreadsFilter;
+import org.apache.lucene.util.VerifyTestClassNamingConvention;
```

Comment:
Hmm, how mysterious, https://github.com/apache/lucene/blob/main/lucene/test-framework/src/java/org/apache/lucene/util/QuickPatchThreadsFilter.java and https://github.com/apache/lucene/blob/main/lucene/test-framework/src/java/org/apache/lucene/util/VerifyTestClassNamingConvention.java are both "neighbours" but somehow there's difficulty discovering `VerifyTestClassNamingConvention` somehow? Had seen this locally but assumed it must be my setup somehow, hmm.

Reply:
Merging in the latest `origin/main` changes somehow solved this, yay!